hypha/apply/activity/templatetags/activity_tags.py

```diff
@@ -49,3 +49,15 @@ def display_for(activity, user):
         return message_data[TEAM]
 
     return message_data[ALL]
+
+
+@register.filter
+def visibility_options(activity, user):
```

Comment:
Can we not reuse the "visibility_choices_for" function in "hypha/apply/activity/models.py"? It has this exact information and I like to avoid duplication.

Reply:
Thanks for the review @frjo! Yes, I totally wanted to avoid duplication. I tried using the visibility_choices_for class method in the template, but Django templates only work with class methods/functions that don't accept params. Making it into a filter is the best way I found to get this logic into the Django template, though I agree with you the duplication isn't at all deal. I'll timebox researching another way to do this, maybe about 30 mins. Tagging @rudietuesdays in case Rae has an idea how to do this!

Reply:
@harumhelmy You can use "visibility_choices_for " directly in the filter like this.